src/test/java/dev/fastgql/DBTestUtils.java

```diff
@@ -72,6 +82,24 @@ public static void executeSQLQueryWithDelay(
             });
   }
 
+  @SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
we should not suppress these warnings, instead it's possible to use "on*" methods and empty subscribe call, or handle returned value

Reply:
I was using it because at line 67 you were also implemented like this

Reply:
right, this needs to be changed as well, I'll take care of this